packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_screen.dart

```diff
@@ -303,3 +467,190 @@ class _DomainCheckTable extends StatelessWidget {
     );
   }
 }
+
+class _PathCheckTable extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final notAvailableCell = DataCell(
+      Text(
+        'Not available',
+        style: TextStyle(
+          color: Theme.of(context).colorScheme.deeplinkUnavailableColor,
+        ),
+      ),
+    );
+    return Opacity(
+      opacity: 0.5,
```

Comment:
do we need the whole table to be wrapped in opacity? It's important to note that rendering Opacity can be expensive.

Reply:
this is also part of the design ,together with the "coming soon"